src/components/PerfilDialog/index.tsx

```diff
@@ -152,7 +152,7 @@ export default function PerfilDialog({ id, readOnly, closeDialog }: PerfilDialog
           <section className="d-flex align-items-center w-100">
             <div className="input-label mt-2"><label className="mr-2">Perfil:</label></div>
             <div className="br-input small">
-              <input data-testid='perfil-nome' className="w-100" placeholder="Digite o nome" value={nome} onChange={e => setNome(e.target.value)} type="text" readOnly={!canEdit} style={{border: !canEdit? "none": ""}} />
+              <input data-testid='perfil-nome' className="w-100" placeholder="Digite o nome do perfil" value={nome} onChange={e => setNome(e.target.value)} type="text" readOnly={!canEdit} style={{border: !canEdit? "none": ""}} />
```

Comment:
A gente poderia pensar em padronizar quebra de linha quando o codigo dos frgament do html (<>...</>) ficar muito longo

```suggestion
              <
              input data-testid='perfil-nome' 
              className="w-100" 
              placeholder="Digite o nome do perfil" value={nome} 
              onChange={e => setNome(e.target.value)} type="text" 
              readOnly={!canEdit} 
              style={{border: !canEdit? "none": ""}} 
              />
```

Reply:
Além disso, não é ideal a cobertura de teste estar abaixo de 80%